src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -23,9 +23,9 @@ interface InputTextProps {
   className?: string;
   value?: string | null;
   placeholder?: string;
-  onChange?: ChangeEventHandler<HTMLInputElement>;
-  onFocus?: FocusEventHandler<HTMLInputElement>;
-  onEnter?: MouseEventHandler<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLElement>;
+  onFocus?: FocusEventHandler<HTMLElement>;
+  onEnter?: MouseEventHandler<HTMLElement>;
```

Comment:
I need a gut check on this.

`HTMLInputElement` allows for value extraction while, I'm pretty sure, `HTMLElement` does not.